rpmlint/checks/PythonCheck.py

```diff
@@ -0,0 +1,64 @@
+import os
+import re
+
+from rpmlint.checks.AbstractCheck import AbstractFilesCheck
+
+# Warning messages
+WARNS = {'tests': 'python-tests-in-package',
```

Comment:
Could we have, please, also a warning against stared `%files` ownership like `%{python_sitelib}/*` or `%{python_sitearch}/*`, which has tendency for package to swallow something which it shouldn't?